live-build/build.gradle

```diff
@@ -113,6 +113,19 @@ for (variant in allVariants) {
                     inputs.property(envVar, System.getenv(envVar)).optional(true)
                 }
 
+                doFirst {
+                    if (!System.getenv('DELPHIX_APPLIANCE_VERSION')) {
+                        /*
+                         * If we are running the build manually and
+                         * DELPHIX_APPLIANCE_VERSION is not set explicitly,
+                         * just use an arbitrarily high value for the
+                         * appliance version to allow upgrading to the
+                         * produced image.
+                         */
+                        environment 'DELPHIX_APPLIANCE_VERSION', '99.9.9.9'
```

Comment:
I think it should be possible to set this in the configuration phase instead of the execution phase, so I don't think this needs to be in a `doFirst` block.

Reply:
Yes, you are probably right. I did this in the execution phase before because it depended on a task being executed. I'll fix that.

Reply:
Can we use the same version scheme as previously used, and what's used by the Jenkins logic? i.e. `"yyyy.M.d.H"`

Reply:
We can, that's what I did in the original review, but the code was more complex. The `"yyyy.M.d.H"` versioning will be deprecated after we transition to the new versioning schema, which will be some time next week.

Reply:
OK. I'm fine with this as-is. Can you point me to any documentation or discussion about this new versioning schema? I don't know anything about that.